src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/custom.py

```diff
@@ -39,8 +39,8 @@ def _update_progress(current, total):
 
 # CUSTOM METHODS
 
-def create_storage_account(resource_group_name, account_name, sku, location=None, kind=None, tags=None,
-                           custom_domain=None, encryption_services=None, access_tier=None, https_only=None,
+def create_storage_account(resource_group_name, account_name, sku='Standard_RAGRS', location=None, kind=None,
```

Comment:
Suggestion:
Leave `sku=None` here and then in _params.py, add a default value for `sku` of `az storage account create` and then get the value from the SDK with `get_sdk(ResourceType.MGMT_STORAGE, 'SkuName')`.

Depending on the value is best as then the SDK is the source of truth.

Reply:
It's still depending on the sdk:
https://github.com/Azure/azure-cli/pull/4723/files#diff-dd843a5a796ae5a954637631e4fcfc40R57

The default value `sku='Standard_RAGRS'` gets passed to `Sku(sku)`. I believe this would be the same as if I set the default in params.

Reply:
I mean `SkuName.standard_ragrs` rather than your string `'Standard_RAGRS'`.
But then you can't make this change in custom.py as you need to conditionally set it if the API version is correct so then it would have to go into `_params.py`

https://github.com/Azure/azure-sdk-for-python/blob/master/azure-mgmt-storage/azure/mgmt/storage/v2017_06_01/models/storage_management_client_enums.py#L25

Reply:
ooh gotchu